plugins/hydra_optuna_sweeper/example/conf/mo-config.yaml

```diff
@@ -0,0 +1,28 @@
+defaults:
+  - override hydra/sweeper: optuna
```

Comment:
since this is a different example, maybe consider separate them into separate folders?

Reply:
I agree.
Can you also add a test covering the new example app (and the existing one if it's not already there)?

Reply:
Thank you for your suggestion. That completely makes sense.
I separated the config directories in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/6103edb13c5dde34f323060d2f311aa06291e9de and added the example app to the test in the commit https://github.com/facebookresearch/hydra/pull/1294/commits/d6286ab9a3801b30376bf34eeb5a726b0de575d8.